static/app/gettingStartedDocs/javascript/nextjs.tsx

```diff
@@ -346,7 +345,7 @@ const feedbackOnboarding: OnboardingConfig = {
         },
       ],
       additionalInfo: (
-        <AdditionalInfoWrapper>
+        <Stack gap="lg">
```

Comment:
`space(2)` is `xl`, not `lg`

```suggestion
        <Stack gap="xl">
```